### DISCUSSION TITLE
Wrong actions dropdown width on spanish version

### DISCUSSION THREAD

**Discussion by andypooletrioteca** [laravel/nova-issues#5510] [Ideas & Feature Requests]
- Laravel Version: 10
- Nova Version: 4.22
- PHP Version: 8.2

### Description:
The spanish word for `Actions` is `Acciones`. It's just a bit longer but it doesn't fit into the actions dropdown, so you get to see `Accione`. I'm struggling to find a way to customize the dropdown width so that the full word fits in.

### Detailed steps to reproduce the issue on a fresh Nova installation:
Translate nova to spanish using https://github.com/coderello/laravel-nova-lang.
Create some actions
Add them to a resource
Navigate to the resource index
Select several resources in the list
Admire the `Accione` dropdown.

**Comment by nordentwickler** [comment]
same for the german translation "Aktionen" - maybe `max-w-[7rem]` is just fine!

**Comment by andypooletrioteca** [comment]
Any updates on this issue? It should be marked as bug, in my opinion, or maybe, if there's an easy way to solve this without waiting for a new nova release, move it to a discussion and throw some light...

**Reply by crynobone** [reply]
We're not considering this a bug and there is no guarantee that the suggested fix will work nicely with all locale.

**Reply by andypooletrioteca** [reply]
You should then consider changing localization documentation as it falsely states:

> Nova may be fully localized
